Plugin/CppApi/source/CoordinateConversionController.cpp

```diff
@@ -330,10 +339,15 @@ void CoordinateConversionController::setInputUtmConversionMode(CoordinateConvers
  */
 void CoordinateConversionController::setPointToConvert(const Esri::ArcGISRuntime::Point& point)
```

Comment:
This is leftover code, but you can remove `Esri::ArcGISRuntime::` here. My bad on that one.